bridge/eth/client_test.go

```diff
@@ -386,8 +432,13 @@ func TestExecute(t *testing.T) {
 			log:        logger.GetOrCreate("testEthClient"),
 		}
 		batch := &bridge.Batch{Id: bridge.NewBatchId(42)}
+		shs := &testsCommon.SignaturesHolderStub{
```

Comment:
maybe make it a static variable in testsCommon since it is written multiple times